test/functional/JIT_Test/playlist.xml

```diff
@@ -523,6 +523,7 @@
 		echo $(Q)$(TEST_JDK_BIN)$(D)jitserver doesn't exist; assuming this JDK does not support JITServer and trivially passing the test.$(Q); \
 	fi; \
 	$(TEST_STATUS)</command>
+		<disabled>Disabled due to a large number of failures which are under investigation.</disabled>
```

Comment:
Would normally put the link to a GitHub issue here so we can track when to reinclude the test.  Suggest creating 1 umbrella issue and relating all other issues to it, so that we have a concrete way of determining when to reinclude the test target.

Reply:
Agreed, thanks.